radio/src/pulses/modules_helpers.h

```diff
@@ -841,7 +841,11 @@ inline void resetMultiProtocolsOptions(uint8_t moduleIdx)
   else {
     g_model.moduleData[moduleIdx].multi.autoBindMode = 0;
   }
+#if defined (RADIO_COMMANDO8)
+  g_model.moduleData[moduleIdx].multi.optionValue = -27;
```

Comment:
What is the purpose of this? this looks like a hack. Please explain.

Reply:
Because our D8 module crystal frequency is 26.000050MHZ, and the R81 receiver crystal frequency is 25.999340MHZ, so there is a frequency deviation and the frequency needs to be compensated. This variable g_model.moduleData[moduleIdx].multi.optionValue cannot save the setting value after switching to another module, so it is set to -27 by default. Is there a better approach for this?

Reply:
Ouch! Ideally this would be fixed with a specific MPM build. I believe this can be set via a `define` in the MPM code. You could then provide specific MPM firmware builds to address this.

Reply:
is this now implemented correctly?

Reply:
Are we talking about frequency tuning for the CC2500 here?? If so, that _normally_ needs to be done on a unit by unit basis anyway. But if you know there is a offset to start with, you can set the forced tuning defines with a build of the MPM firmware and set the default values at the MPM firmware level. https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/docs/Frequency_Tuning.md#forced-tuning-values

Reply:
Yes, it's a CC2500 module. We'll checkout the link. thanks

Reply:
We fixed the code of our module. No EdgeTX code adaption necessary anymore.